### ISSUE TITLE
Report not clear when using --no-delete

### ISSUE THREAD

**Issue by MauroSteel** [siom79/jdrivesync#17]
Using the command
`    /opt/jdrivesync/jdrivesync.sh --down --no-delete -l "/home/mrfoobar/cloud/test" -r Publicaties/Inboxdocument
`
works as desired, but this testcase generated report below. Why are the files reported as deleted, when nothing actually is deleted? In this testcase it's about 10 files I move around. In a live-situation it has to work with more files. 

2017-07-19 15:28:41 INFO jdrivesync.report.NoopReport log(): Synchronized Deleted /home/mrfoobar/cloud/test/document 2017 - 14.pdf
2017-07-19 15:28:41 INFO jdrivesync.report.NoopReport log(): Synchronized Deleted /home/mrfoobar/cloud/test/document 2017 - 21.pdf
2017-07-19 15:28:41 INFO jdrivesync.report.NoopReport log(): Synchronized Deleted /home/mrfoobar/cloud/test/document 2017 - 28.pdf
2017-07-19 15:28:41 INFO jdrivesync.report.NoopReport log(): Synchronized Deleted /home/mrfoobar/cloud/test/document 2017 - 7.pdf
2017-07-19 15:28:41 INFO jdrivesync.report.NoopReport log(): Synchronized Deleted /home/mrfoobar/cloud/test/architecture-2083687_1280.jpg
2017-07-19 15:28:41 INFO jdrivesync.report.NoopReport log(): Synchronized Deleted /home/mrfoobar/cloud/test/archway-700875_640.jpg
2017-07-19 15:28:41 INFO jdrivesync.report.NoopReport log(): Synchronized Deleted /home/mrfoobar/cloud/test/cp-kn_pdf_2017_no._14.pdf
2017-07-19 15:28:41 INFO jdrivesync.report.NoopReport log(): Synchronized Deleted /home/mrfoobar/cloud/test/lvdk2013-745.pdf
2017-07-19 15:28:41 INFO jdrivesync.report.NoopReport log(): Synchronized Deleted /home/mrfoobar/cloud/test/red.txt
2017-07-19 15:28:41 INFO jdrivesync.report.NoopReport log(): Synchronized Deleted /home/mrfoobar/cloud/test/reed-546250_640.jpg
2017-07-19 15:28:41 INFO jdrivesync.report.NoopReport log(): Synchronized Created /e.l._213.pdf
2017-07-19 15:28:42 INFO jdrivesync.App printStatistics(): Statistics:
2017-07-19 15:28:42 INFO jdrivesync.App printStatistics(): NEW:       1 file
2017-07-19 15:28:42 INFO jdrivesync.App printStatistics(): DELETED:   10 files
2017-07-19 15:28:42 INFO jdrivesync.App printStatistics(): UPDATED:   0 files
2017-07-19 15:28:42 INFO jdrivesync.App printStatistics(): UNCHANGED: 0 files

**Comment by siom79** [comment]
Is the directory `/home/mrfoobar/cloud/test` empty when you start jdrivesync? Do these 10 files exist in your Google Drive?

**Comment by MauroSteel** [comment]
No, the 10 files where in this test-folder before i synced. As said, it works as expected. But the warning isn't clear. Should this not be 'ignored' or something?
